src/FluentCMS/Templates/Admin/manifest.json

```diff
@@ -161,6 +188,26 @@
             "Path": "/admin",
             "Plugins": [],
             "Children": [
+                {
+                    "Title": "Content Management",
+                    "Path": "/content",
+                    "Plugins": [
+                        {
+                            "Definition": "Content Management",
+                            "Section": "Main"
+                        }
+                    ]
+                },
+                {
+                    "Title": "Content Management",
```

Comment:
@ParsaGachkar How to create dynamic pages?

`/admin/content/posts` => `/?pluginDef=Content%20Management&typeName=Content%20List&contentTypeSlug=posts`

Reply:
Manifest.json is not responsible for creating dynamic pages. It's only responsible for creating initial data. ContentType api should manage creation and modification of pages in database